actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkMigrationActor.java

```diff
@@ -273,6 +375,44 @@ private String getCreatedBy(Request request) {
     return mappedColumns;
   }
 
+  private List<String> mapSelfDeclaredCsvColumn(List<String> csvColumns) {
+    List<String> mappedColumns = new ArrayList<>();
+    csvColumns.forEach(
+        column -> {
+          if (column.equalsIgnoreCase("email id")) {
```

Comment:
@reshmi-nair @Hari-stackroute - we can't afford to have the column names here as a string. These declarations can be different for personas. We could use tenant pref APIs and get them back.

Reply:
@indrajra :  will take this in next release(3.3.0)

Reply:
We discussed - must be in 3.2 itself